app/javascript/packs/intl-tel-input.js

```diff
@@ -1,38 +1,16 @@
+import { loadPolyfills } from '@18f/identity-polyfill';
 import 'intl-tel-input/build/js/utils.js';
 import * as intlTelInput from 'intl-tel-input/build/js/intlTelInput';
 
-const telInput = document.querySelector('#new_phone_form_phone');
-const intlCode = document.querySelector('#new_phone_form_international_code');
-
-// initialise plugin
-intlTelInput(telInput, {
-  preferredCountries: ['US', 'CA'],
-  onlyCountries: JSON.parse(intlCode.getAttribute('data-countries')),
-});
-
-// OnChange event
-telInput.addEventListener('countrychange', function () {
-  const selected = document.querySelector(".iti__country[aria-selected='true']");
-  const country = selected.getAttribute('data-country-code').toUpperCase();
-  // update international_code dropdown
-  for (let i = 0; i < intlCode.options.length; i += 1) {
-    if (intlCode.options[i].value === country) {
-      intlCode.options[i].selected = true;
-    }
-  }
-  // Using plain JS to dispatch the country change event to phone-internationalization.js
-  telInput.dispatchEvent(new Event('countryChange'));
-});
-
 function intlTelInputNormalize() {
   // remove duplacte items in the country list
```

Comment:
As long as we're here, can we fix the typo?

```suggestion
  // remove duplicate items in the country list
```

Reply:
> As long as we're here, can we fix the typo?

👍  Updated in b84714b.